src/probability/strong_law.lean

```diff
@@ -0,0 +1,752 @@
+/-
+Copyright (c) 2022 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel
+-/
+
+import probability.ident_distrib
+import measure_theory.function.l2_space
+import measure_theory.integral.interval_integral
+import analysis.specific_limits.floor_pow
+import analysis.p_series
+import analysis.asymptotics.specific_asymptotics
+
+/-!
+# The strong law of large numbers
+
+We prove the strong law of large numbers, in `probability_theory.strong_law_ae`:
+If `X n` is a sequence of independent identically distributed integrable real-valued random
+variables, then `∑ i in range n, X i / n` converges almost surely to `𝔼[X 0]`.
+We give here the strong version, due to Etemadi, that only requires pairwise independence.
```

Comment:
Could you please reference the Etemadi's paper? I guess, [this one](https://link.springer.com/article/10.1007/BF01013465)?